infrastructure/variables.tf

```diff
@@ -1,6 +1,5 @@
 variable "product" {
   type    = "string"
-  default = "draft-store"
```

Comment:
is this related?

Reply:
Possibly. After the first commit terraform complained about not being able to find the server farm and as far as I remember that had something to do with setting the product name in code (might be wrong). This default is unnecessary, anyway.

Reply:
yeah it shouldn't be in here, it comes from the jenkinsfile now

Reply:
I've changed the title of the PR to be more accurate.